_i18n/en.yml

```diff
@@ -147,42 +147,50 @@ publications-tweet_feed:
            <p>We can help you start a new project, instil XP practices or Devops culture into your existing teams, plan your architectural evolution or adopt a new technology.</p>
            <p>We are available to run workshops and engage with you on a short-term basis to ensure that you put your best foot forward for your next initiative.</p>
 services-expert_advice-enterprise_architecture:
    title: Enterprise Architecture
-   subtitle: <p>Our Craftsmen have a wealth of expertise and experiences in dealing with large scale systems in finance, media and government.</p>
-           <p>We will review your approach and help you make the most effective decisions for the future.</p>
+   subtitle: Our Craftsmen have a wealth of expertise and experiences in dealing with large scale systems in finance, media and government. We will review your approach and help you make the most effective decisions for the future.
 services-expert_advice-software_architecture_and_design:
    title: Software Architecture & Design
-   subtitle: <p>We will review your current architecture and help you create a risk-based view of your software solution. We can advise on the best way to evolve your architecture to reduce those risks.</p>
-           <p>If you are looking to evolve your system into microservices or create a microservices based architecture from scratch, we will help you understand the best approach according to your needs.</p>
+   subtitle: We will review and help create an apporach to evolve your architecture towards an agreed vision. We have deep knowledge and experience in cloud technologies and enterprise architectures, including Domain Driven Design, Microservices, Event Sourcing and Serverless.
 services-expert_advice-continous_delivery:
    title: Continuous Delivery
-   subtitle: We are experts in creating continuous delivery pipelines. We understand that it is not only a technical challenge but also an organisational one. We help your customers evolve their approach towards continuous delivery aligning their business, teams and technology.
+   subtitle: We are experts in creating continuous delivery pipelines. We understand that it is not only a technical challenge but also an organisational one. We help our customers evolve their approach towards continuous delivery - aligning their business, teams and technology.
 services-expert_advice-xp:
-   title: Extreme Programming Practices
-   subtitle: XP practices are core to every Craftsman in Codurance. We love to coach and mentor our customers in adopting XP practices. We advise you on how to adopt the practices to bring about positive change in your teams.
+   title: Extreme Programming (XP) Practices
+   subtitle: XP practices are core to every Craftsman in Codurance. We love to coach and mentor our customers in adopting these. We advise on how to adopt XP practices and Softeware Craftsmanship values to bring about positive change in software development teams.
```

Comment:
XP practices are core to everyone at Codurance. We love to coach and mentor our customers in adopting them. We advise on how to adopt XP practices and Software Craftsmanship values to bring about positive change in software development teams.